map/framework.cpp

```diff
@@ -169,6 +170,38 @@ void CancelQuery(weak_ptr<search::ProcessorHandle> & handle)
     queryHandle->Cancel();
   handle.reset();
 }
+
+string GetStreet(search::ReverseGeocoder const & coder, Index const & index, FeatureType const & ft)
```

Comment:
Please, remove const from the FeatureType. GetCenter() actually modifies the contents of the FeatureType.